style.css

```diff
@@ -0,0 +1,744 @@
+/*
+black: #272a31
+orange: #ec5242
+silver: #d3d3d3
+font-family: cocogoose, lato */
+body {
+  font-family: 'Lato', sans-serif;
+  color: #272a31;
+}
+
+.flex {
+  display: flex;
+}
+
+/*********** trial */
+
+/* Mobile Menu */
+.mobile-menu {
+  display: none;
+  flex-direction: column;
+  background: #272a31;
+  opacity: 95%;
+  width: 98vw;
+  height: 86.8vh;
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+.menu-div {
+  display: flex;
+  flex-direction: column;
+}
+
+.mobile-menu div a {
+  font-weight: 600;
+  font-size: 2rem;
+  line-height: 7vh;
+  color: #ec5242;
+  text-decoration: none;
+  padding: 3vh;
+  margin: 0;
+}
+
+.menu-div a:hover {
+  text-decoration: underline;
+}
+
+.close-item {
+  color: white;
+  align-self: flex-end;
+  padding-top: 3vh;
+  padding-right: 4vw;
+}
+
+/* Header Section Mobile */
+.header-first-line,
+.header-second-line,
+.header-logo,
+.program-desktop {
+  display: none;
+}
+
+.header-nav {
+  padding: 2vh 0 4vh 2vw;
+}
+
+.header-content {
+  background-image: url(images/mainbackground.png);
+
+  /* background-size: auto;
+  background-repeat: no-repeat; */
+  padding: 1vh 4vw 9vh 4vw;
+}
+
+.header-content p {
+  color: #ec5242;
+  font-weight: 400;
+}
+
+.header-content h1 {
+  color: #ec5242;
+  font-family: 'COCOGOOSE', sans-serif;
+}
+
+.header-paragraph {
+  border: 3px solid white;
+  padding: 2vh 2vw;
+  line-height: 170%;
+  background: #d3d3d3;
+}
+
+.header-content h2 {
+  color: #3b3f47;
+  font-size: 1.2rem;
+  font-family: 'COCOGOOSE', sans-serif;
+  font-weight: bold;
+}
+
+/* Program Section Mobile */
+.program {
+  background-image: url(images/programbackground.png);
+
+  /* display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr; */
+  gap: 2.5vh;
+  padding: 3vh 2vw 8vh 2vw;
+  color: #d3d3d3;
+}
+
+.program-h2,
+.partners-h2,
+.about-us-h2,
+.speak-h2 {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.program-h2 h2,
+.partners-h2 h2,
+.about-us-h2,
+.speak-h2 {
+  align-self: center;
+}
+
+.horizontal-hr {
+  border-width: 0;
+  height: 2px;
+  background-color: #ec5242;
+  width: 15%;
+}
+
+/*
+.lecture {
+  background: #272a31;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  height: auto;
+  opacity: 0.7;
+}
+
+.lecture-left {
+  display: flex;
+  width: 70%;
+  align-items: baseline;
+}
+
+.lecture-left i {
+  padding: 0 2vw;
+}
+
+.lecture-left h3 {
+  color: #ec5242;
+  font-size: 1rem;
+}
+
+.lecture-right {
+  display: flex;
+}
```

Comment:
- [ ] There are so many commented code blocks in the stylesheet. Please check thoroughly and remove them all.

Reply:
Alright, thank you!

Reply:
This issues is not properly resolved. 😿